code/datums/diseases/advance/symptoms/emp.dm

```diff
@@ -0,0 +1,45 @@
+/datum/symptom/emp
+	name = "Electromagnetic Pulse"
+	desc = "The virus occasionally emits an electromagnetic pulse, disabling electronics in the host's inventory and on their tile."
+	stealth = 0
+	resistance = 1
+	stage_speed = 1
+	transmittable = 1 //eh, screw it, these stats will do
```

Comment:
net positive stats for something you can append onto naughty diseases, effectively btfo'ing making cures in either the chemdisp or pandemic 😏